Earthfile

```diff
@@ -300,6 +300,7 @@ base-image:
     ARG KAIROS_VERSION
     ARG BUILD_INITRD="true"
     ARG TARGETARCH
+    ARG HWE
```

Comment:
What are the 2 possible values?

`-hwe` and `-non-hew` ? Since there is no use of this yet in the pipelines, it would be useful to add a comment with the possible values.